### DISCUSSION TITLE
a simple question about...

### DISCUSSION THREAD

**Discussion by 62Maurizio** [keirf/flashfloppy#849] [General]
Hi...
Env : Host ms-dos 6.22 , device gotek board sfrkc30.at2
Act: i successfully format a pen to emulate 100 drives.. 
but when i put the pen into gotek, pressing swithces the display does not show the drives number just formatted ,,,
When i boot pc it recognize drive A, but when i try to access it, the pc tell me;  Drive not ready
In your opinion, is this a firmware error or a different hardware error, or i do something wrong during installation 
Thanks in advance for your support

**Comment by keirf** [comment]
FlashFloppy doesn't use 100 partitions like factory firmware. Instead format the whole usb stick as FAT32. Then use image files (suffix IMG) of size 720kb and 1440kb.

**Comment by 62Maurizio** [comment]
Hi.. 
Thanks for the quick answer.. and forgive me if i bother you, but i'm are a step backward...
i buy the d gotek and i'm unable to make it function, the gotek is installed on a pc  that drive a cnc machine... i want to substitute diskettes..
this machine is able to read normal floppy, so i installed gotek machine... at startup dos 6.22 recognize the drive, but when i try to issue A: 
it display a drive not ready msg... 
i suspected an hardware fail, but i do not know if related to firmaware or phisical component, so before buy another gotek i want to clear my ideas.. the chipset inside card sfrkc0.at2 is a QFN32F415 ... now the question is ; i have an harware error? So i have to buy a new one, or a firmaware update will fix the problem ? i know is not you work to perform problem determination, it is just a simple question , that will help me a lot, because this is my first experience with a gotek device, and viceversa you are very experienced with thia device
Thanks in advance
Maurizio
